releases/0.4.0/index.md

```diff
@@ -4,16 +4,58 @@ layout: releases
 custom_css: releases
 ---
 
-## {{ page.title }}
+# {{ page.title }}
 
-###### Date: _TBD_
+##### Date: 2021-05-27
 
-#### Ontology File(s)
+## Ontology File(s)
 
-[GitHub](https://github.com/casework/CASE/tree/develop)
+[GitHub](https://github.com/casework/CASE/releases/tag/0.4.0)
 
-#### Migration guidance
+## Migration guidance
 
 A guide to migration from CASE 0.3.0 will be provided.
```

Comment:
The changes solely done in CASE are only additive, so there is no migration guidance for CASE concepts, and we should note this.

There is one migration point we should document of a UCO concept that UCO 0.6.0's release notes don't document.  `observable:contactName` no longer exists, and data held in that predicate can now go to either `observable:displayName` or `observable:nickname`, as the user or tool finds appropriate.

Reply:
Added as part of [PR 108](https://github.com/casework/casework.github.io/pull/187).